templates/apiary-bucket-policy.json

```diff
@@ -150,6 +150,50 @@
             ]
         },
 %{endif}
+%{if common_producer_iamroles != ""}
+        {
+            "Sid": "General read-write iamrole permissions",
+            "Effect": "Allow",
+            "Principal": "*",
+            "Action": [
+                "s3:GetAccountPublicAccessBlock",
```

Comment:
Are all of these permissions needed?
The definition only mentions read and write permissions to buckets.

Reply:
Let me adjust the permissions to match a subset of the permissions in producer_iamroles.